config/locales/en.yml

```diff
@@ -1492,23 +1468,21 @@ en:
         body: |
           Hello <<Client.PreferredName>>,
 
-          Your tax information has been successfully submitted! Your tax specialist will review your information and reach out if additional information is needed before preparing your taxes.
+          Your tax information has been successfully submitted! Your Client ID is <<Client.ClientId>>. This is an important piece of account information that can assist you when talking to our chat representatives and when logging into your account. Please keep track of this number and do not delete this message.
+
+          Your tax specialist will review your information and reach out if additional information is needed before preparing your taxes.
 
           Check your progress and add additional documents here: <<Client.LoginLink>>
 
           If you have any questions, please reply to this message.
 
           We’re here to help!
-
-          Your tax team at GetYourRefund.org
-        subject: Your tax information has been successfully submitted!
+          Your tax team at GetYourRefund
+        subject: Welcome to GetYourRefund/ Client ID
       sms: |
-        Hello <<Client.PreferredName>>,
-        Your tax information has been successfully submitted! Your tax specialist will review your information and reach out in 3-5 business days for a phone review of your situation before preparing your taxes.
-        Check your progress and add additional documents here: <<Client.LoginLink>>
-        If you have any questions, please reply to this message.
-        We’re here to help!
-        Your tax team at GetYourRefund.org
+        Hello <<Client.PreferredName>>, Your tax information was successfully submitted to GetYourRefund**.** IMPORTANT**:** Your Client ID is <<Client.ClientId>>. Please keep track of this number and do not delete this message.
```

Comment:
In terms of English punctuation/spacing, I think this should be `**IMPORTANT**` not `** IMPORTANT**`.

However, are you sure program/product wants these asterisks here? I don't see them in the Notion doc.